### ISSUE TITLE
No Overview Page for Backend

### ISSUE THREAD

**Issue by kryptoblack** [hashicorp/terraform#34479]
### Terraform Version

```shell
Terraform v1.6.6
on linux_amd64
```

### Affected Pages

https://developer.hashicorp.com/terraform/language/v1.1.x/settings/backends

### What is the docs issue?

If you visit the above link you would find the Backend Overview Page for version `v1.1 and earlier`. If you then try to change the version to `v1.2.x`  you would notice that the **Terraform Settings > Backends > Overview** has disappeared from the sidebar. If you then try to select any other version apart from the ones already mentioned you would arrive at a **Page Not Found** error.

Another issue is that this page has been referenced in **[Terraform Study Guide > Manage State](https://developer.hashicorp.com/terraform/tutorials/certification-003/associate-study-003#manage-state) > Backend management**

### Proposal

Add Overview page for Backend for version `1.2.x` and above. 

There are other ways to solve this issue but I would like to understand what might be the intended behavior.

**Comment by apparentlymart** [comment]
Thanks for reporting this, @kryptoblack.

If I recall correctly, it was intentional to merge the overview page with the configuration page to produce what is currently [Backend Configuration](https://developer.hashicorp.com/terraform/language/settings/backends/configuration).

However, when we do such things we typically add a redirect from the old location, which seems like it was missed here. If so, I imagine the fix here will be to add that redirect so that the old URL of the overview page redirects to the `configuration` page.

It's also strange that the guide you mentioned is linking directly to the archived v1.1.x documentation rather than to latest. Those guides live in a non-public repository and so that would need to be corrected in a different place; hopefully whoever on the docs team ends up fixing the missing redirect (assuming I'm right about that) can also find the source of that tutorial and update it to link to the non-version-specific URL.

Thanks again!

**Comment by kryptoblack** [comment]
If that's the case, you might want to remove the link entirely as there is already a link to Backend Configuration.

Also, if the link to archive docs is unexpected behaviour you might want to check other links on the Study Guide as a lot of them refer to `1.1.x`

**Comment by apparentlymart** [comment]
Indeed... unless a tutorial is specifically about the behavior of Terraform v1.1 or earlier (or any other specific version) then it should link to the unversioned URL.

A guide specifically about Terraform v1.1 would be very surprising at this point, because that series is _long_ obsolete! Therefore I would guess this was probably an editing error, accidentally copying a versioned URL instead of an unversioned one when preparing the tutorial.
